recipes/tabulate/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, tools
+
+
+class Tabulate(ConanFile):
+    name = "tabulate"
+    
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/p-ranav/tabulate"
+    description = "Table Maker for Modern C++"
+    author = "Pranav"
+    topics = ("conan", "cpp17", "table", "table-maker", "formatted-text", "terminal", "cli")
+    license = "MIT"
+    no_copy_source = True
+    settings = "compiler"
+    
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "17")
```

Comment:
@jgsogo The problem is that there is no compiler which supports C++17 by default. As compiler.cppstd is not declared, Conan will use the default C++ version supported by the compiler. GCC9, for instance, is C++14.

Related issue: https://github.com/conan-io/conan/issues/6282